less/ksf-grid.less

```diff
@@ -0,0 +1,9 @@
+@import (reference) '../less/ksf-variables';
+
+.grid--row {
```

Comment:
Empty rule?

Reply:
There will probably be some rules at some point, although for now it can be removed